src/ripple/protocol/impl/STVar.h

```diff
@@ -54,12 +57,12 @@ class STVar
 
     STVar (STBase&& t)
     {
-        p_ = t.move(sizeof(d_), &d_);
+        p_ = t.move(max_size, &d_);
```

Comment:
Hmmmm.  I wonder if using `max_size` here rather than `size_of(d_)` is a pessimization.  Since I was unacquainted with `aligned_storage` I ran an experiment in Wandbox.  Running gcc in Wandbox reports the actual size of `d_` is 80.  So, at least in gcc, we could be wasting 8 bytes of storage per `STVar`.  Clearly using `max_size` is also safe, so it's not really a bug, but it may be less efficient.

I'm not acquainted with `aligned_storage`.  I'm assuming the compiler will obey the alignment requirement if an `STVar` is created on the stack?

Just curious if anyone knows why `72` was picked.  If we do know, it would be good to leave a comment.